app/components/sign-up/form.hbs

```diff
@@ -0,0 +1,56 @@
+{{#if @changeset.errors}}
```

Comment:
I think there needs to be a similar title here like the dj sign up form. maybe also some text below to explain what they are signing up for